eslint-example/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+		"globals": {"chrome": true},
+    "rules": {
+        "no-set-state": "off"
+    },
+    "env": {
+    		"browser": true,
+        "es6": true
+    }
```

Comment:
I think it's worth add [extends: "eslint:recommended"](https://github.com/Standard8/example-webextension/blob/master/.eslintrc.js#L19) because that catches a lot of standard JS errors.